### PR TITLE
Doc: Find hostname of a floating IP

### PR DESCRIPTION
**Pull request by feber**
Additional information for cPouta users, on how they can find hostname of a floating IP.

Edit: [link to section](https://docs.csc.fi/cloud/pouta/additional-services/#dns-services-in-cpouta).

### CODE REVIEW

File: docs/cloud/pouta/additional-services.md [hunk 1]
```diff
@@ -38,12 +38,25 @@ the CSC Service Desk to coordinate this.
...
 cPouta does not currently offer integrated name service management.
 
-All our floating IPs are mapped to a default hostname, for example
+All our floating IPs are mapped to a default hostname, for example:
 
 ```
 vm0120.kaj.pouta.csc.fi
 ```
 
+To find hostname of a floating IP, you could leverage `host` command:
```
Comment (matteopozza) [inline docs/cloud/pouta/additional-services.md:47]: You can add "the" in front of "hostname" -> "To find _the_ hostname of a floating IP..."

I would also rephrase "you could leverage host command" to a simpler "you can use the host command" or "you can leverage the host command"

File: docs/cloud/pouta/additional-services.md [hunk 2]
```diff
@@ -38,12 +38,25 @@ the CSC Service Desk to coordinate this.
...
+```sh
+host -a <floating IP address>
+```
+
+And the result is `vmXXXX.kaj.pouta.csc.fi` listed in output:
+
+```
+... omitted for brevity
```
Comment (matteopozza) [inline docs/cloud/pouta/additional-services.md:55]: I think you can remove this line. Alternatively, you can just keep the three dots, i.e., without the "omitted for brevity". In my opinion, the three dots already indicate that something else is expected before the subsequent lines, so they are sufficient.

File: docs/cloud/pouta/additional-services.md (hunk 1 above)
Comment (feber) [reply docs/cloud/pouta/additional-services.md]: addressed, thanks

File: docs/cloud/pouta/additional-services.md (hunk 2 above)
Comment (feber) [reply docs/cloud/pouta/additional-services.md]: fair enough. yes I agree that `omitted for brevity` here is kind of duplicate. thanks.

**Review by matteopozza — APPROVED** [review]
This is a nice contribution to our documentation, thanks!

**Pull request merged**